### PR TITLE
Tooltip highlights

### PR DESCRIPTION
**Pull request by cjquines**
conflicts with #1257 and i prefer that get merged first

also adjusts threshold for search results

image

### CODE REVIEW

File: static/keyboard.js [hunk 1]
```diff
@@ -211,46 +236,53 @@ function Keyboard() {
...
+          return map;
+        }, new Map())
+        .entries(),
+    ];
+    return keys.map(([i, results]) => {
       const chr = codepage[i];
       return html`<${Tooltip}
         key=${i}
-        shown=${shownIndexes.includes(i)}
+        shown=${true}
```
Comment (cjquines) [inline static/keyboard.js:282]: oh lol i should remove this

Comment (Steffan153) [reply static/keyboard.js:282]: often when I'm too lazy to open up vscode I just slam suggestions at myself
```suggestion
        shown=${shownIndexes.includes(i)}
```

Comment (cjquines) [reply static/keyboard.js]: no i mean to just remove the argument lol

**Review by Steffan153 — APPROVED** [review]
Cool
image

**Review by chunkybanana — APPROVED** [review]
This looks like it works, it looks really cool, and I can't be bothered testing it.

**Pull request merged**